### DISCUSSION TITLE
Integrare il componente

### DISCUSSION THREAD

**Discussion by Dany84L** [jasonacox/tinytuya#217] [General]
Buon Pomerioggio.
Come faccio a integrare il componente TinyTuya senza utilizzare python?
Lovorrei integrare con indirizzo web direttamente da HACS ma con il codice che c'è su Github mi dice che non è completo.
Grazie mille per l'aiuto.

**Comment by jasonacox** [comment]
Ciao amico, HA (home assistant) usa Python, ma TinyTuya non è progettato per essere un plugin per HA. Ci vorrebbe un po' di lavoro per renderlo compatibile. Grazie per aver sollevato questo problema. Se c'è abbastanza interesse e supporto da parte della comunità, possiamo dare un'occhiata a cosa ci vorrebbe per renderlo un plugin.

> Hi Friend, HA (home assistant) uses python, but TinyTuya is not designed to be a plugin for HA. It would require some work to make it compatible.  Thanks for raising this. If there is enough interest and community support, we can take a look at what it would take to make it a plugin.

**Comment by Dany84L** [comment]
Grazie per la risposta. Python lo devo installare su home assistant? Ho provato ma. Non riesco. C'è una guida specifica di come fare? Grazie

Ottenere Outlook per Android<https://aka.ms/AAb9ysg>
________________________________

**Reply by jasonacox** [reply]
Non sono un esperto di HA, ma questo potrebbe aiutare: https://www-home--assistant-io.translate.goog/integrations/python_script/?_x_tr_sl=auto&_x_tr_tl=it&_x_tr_hl=it&_x_tr_pto=wapp

> I'm not a HA expert but this might help: https://www-home--assistant-io.translate.goog/integrations/python_script/?_x_tr_sl=auto&_x_tr_tl=en&_x_tr_hl=en&_x_tr_pto=wapp

**Comment by Dany84L** [comment]
Grazie. Molto gentile. Buona giornata

Ottenere Outlook per Android<https://aka.ms/AAb9ysg>
________________________________
